terraso_backend/apps/graphql/schema/landscapes.py

```diff
@@ -65,17 +70,82 @@ class Meta:
             "partnership_status",
             "profile_image",
             "profile_image_description",
+            "center_coordinates",
         )
 
         interfaces = (relay.Node,)
         connection_class = TerrasoConnection
 
     area_scalar_ha = graphene.Float()
 
+    @classmethod
+    def get_queryset(cls, queryset, info):
+        # Prefetch default landscape group, account membership and count of members
+
+        is_anonymous = info.context.user.is_anonymous
+        group_queryset = (
+            Group.objects.prefetch_related(
+                # Prefetch account membership
+                Prefetch(
+                    "memberships",
+                    to_attr="account_memberships",
+                    queryset=Membership.objects.filter(
+                        user=info.context.user,
+                        membership_status=Membership.APPROVED,
+                    ),
+                ),
+            )
+            if not is_anonymous
+            else Group.objects.all()
+        )
+
+        try:
+            # Fetch all fields from Landscape, except for area_polygon
+            exclude_fields = ["area_polygon"]
+            include_fields = [
+                f.name for f in Landscape._meta.get_fields() if f.name not in exclude_fields
+            ]
+
+            result = (
+                queryset.only(*include_fields)
+                .prefetch_related(
+                    Prefetch(
+                        "associated_groups",
+                        to_attr="default_landscape_groups",
+                        queryset=LandscapeGroup.objects.prefetch_related(
+                            Prefetch(
+                                "group",
+                                queryset=group_queryset.annotate(
+                                    memberships_count=Count(
+                                        "memberships__user",
+                                        distinct=True,
+                                        filter=Q(
+                                            memberships__membership_status=Membership.APPROVED
+                                        ),
+                                    )
+                                ),
+                            ),
+                        ).filter(is_default_landscape_group=True),
+                    )
+                )
+                .all()
+            )
+        except Exception as e:
+            logger.exception("Error prefetching Landscape data", error=e)
+            raise e
+        return result
+
     def resolve_area_scalar_ha(self, info):
         area = self.area_scalar_m2
         return None if area is None else round(m2_to_hectares(area), 3)
 
+    def resolve_default_group(self, info):
```

Comment:
question: is this the default group for a landscape?

Reply:
@paulschreiber That is the Graph QL resolver that will send the landscape default group to the defaultGroup query field, so it is the function that will return the default group, let me know if that makes sense